### PR TITLE
ttf2pt1: fix include the FreeType front-end

### PR DESCRIPTION
**Pull request by pklaus**
ttf2pt1's important FreeType front-end was left out up to now. I changed the formula to include it.

The commit also includes a [necessary patch from macports](https://trac.macports.org/browser/trunk/dports/print/ttf2pt1/files/patch-ft.c?rev=144190) at the end of the formula.

### CODE REVIEW

File: Library/Formula/ttf2pt1.rb [hunk 1]
```diff
@@ -4,9 +4,43 @@ class Ttf2pt1 < Formula
   url "https://downloads.sourceforge.net/ttf2pt1/ttf2pt1-3.4.4.tgz"
   sha256 "ae926288be910073883b5c8a3b8fc168fde52b91199fdf13e92d72328945e1d0"
 
+  patch :DATA
```
Comment (mikemcquaid) [inline Library/Formula/ttf2pt1.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

Comment (pklaus) [inline Library/Formula/ttf2pt1.rb]: Unfortunately this is hopeless. The last release of the tool was in 2003 and unmaintained now:
<http://ttf2pt1.sourceforge.net/download.html>

Without the patch I get the error `/usr/local/include/freetype/freetype.h:24:2: error: "'ft2build.h' hasn't been included yet!"`. The patch is needed to fix the freetype headers for its changed API and v2.

Comment (mikemcquaid) [inline Library/Formula/ttf2pt1.rb]: I'd probably recommend we don't add freetype as an option, then.

**Comment by mikemcquaid** [comment]
If this is software is now unmaintained we may want to consider moving it to the boneyard.

**Comment by pklaus** [comment]
> If this is software is now unmaintained we may want to consider moving it to the boneyard.

If have no strong opinion on it as I don't know your current policy concerning unmaintained software.

But this tool is still being used by a lot of software in the typesetting community on Unix / Linux / OS X. They have a different pace for switching to different tools and deprecating others.

**Pull request closed without merging**